wagtailio/static/sass/components/_buttons.scss

```diff
@@ -4,7 +4,6 @@
     text-decoration: none;
     border-radius: 6px;
     border: 1px solid $color--off-black;
-    margin: 0 auto;
```

Comment:
Having the buttons always align themselves could cause us extra work removing margins in the future

Reply:
I agree. I think it was added so the buttons display normally [in the pattern library](http://localhost:8000/pattern-library/pattern/patterns/styleguide/buttons.html). We could add an inline `<style>` tag on the [buttons PL template](https://github.com/wagtail/wagtail.org/blob/next/wagtailio/project_styleguide/templates/patterns/styleguide/buttons.html) and move that rule here so they're not full width in the PL

Reply:
Awesome, that makes sense. I'll do that now :)